changelog.d/20240322_122646_derek_non_experimental_scope_parsing_sc_30765.rst

```diff
@@ -0,0 +1,6 @@
+
+Added
+~~~~~
+
+- Moved Scope parsing out of experimental. The `Scope` construct is not importable from
```

Comment:
We could possibly add more here (e.g. example usage in the readme), but I think these minor adjustments are necessary:

```suggestion
- Moved scope parsing out of experimental. The ``Scope`` construct is now importable from
```